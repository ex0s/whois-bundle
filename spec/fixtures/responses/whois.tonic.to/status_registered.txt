Tonic whoisd V1.1
go ns-1.myphotoalbum.com
go ns-2.myphotoalbum.com
