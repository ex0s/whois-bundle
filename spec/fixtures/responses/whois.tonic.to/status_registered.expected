#disclaimer
  %s %ERROR{AttributeNotSupported}


#domain
  %s %ERROR{AttributeNotSupported}

#domain_id
  %s %ERROR{AttributeNotSupported}


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %ERROR{AttributeNotSupported}

#updated_on
  %s %ERROR{AttributeNotSupported}

#expires_on
  %s %ERROR{AttributeNotSupported}


#registrar
  %s %ERROR{AttributeNotSupported}

#registrant_contacts
  %s %ERROR{AttributeNotSupported}

#admin_contacts
  %s %ERROR{AttributeNotSupported}

#technical_contacts
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %ERROR{AttributeNotSupported}


#response_incomplete?
  %s == false
